### PR TITLE
Updated versioning allows java 13+ to be used when compiling

### PR DESCRIPTION
**Pull request by Tim-Ganger**
These changes fixed version incompatibilities and removed depreciated syntax, imports, and classes.

### CODE REVIEW

File: .idea/checkstyle-idea.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcoblenz) [inline .idea/checkstyle-idea.xml:1]: Is this file required, or is it really a separate change?

File: buildscript/build.gradle [hunk 2]
```diff
@@ -37,6 +37,7 @@ dependencies {
     compile 'org.glassfish:javax.json:1.1.0-M1'
     compile 'org.json:json:20171018'
     compile 'com.google.protobuf:protobuf-java:3.11.4'
+    compile ''
```
Comment (mcoblenz) [inline buildscript/build.gradle:40]: Remove?

File: .idea/checkstyle-idea.xml (hunk 1 above)
Comment (Tim-Ganger) [reply .idea/checkstyle-idea.xml:1]: I don't think so, it looks like this was created by intellij and I didn't filter it out.
